Documentation/cubes.rst

```diff
@@ -224,17 +224,44 @@ These primitives allow to associate transparent cubes to standard cube's ports o
   polycubectl attach firewall0 veth1
 
 
-Span Mode
----------
+Hook points
+-----------
 
-The shadow cubes have a mode called **span**.
+Differently from traditional eBPF programs, a Polycube service is created in a way that works when attached to either ``TC``, ``XDP_SKB`` or ``XDP_DRV`` eBPF hooks.
+Hence, you can attach aPolycube service to either the ``TC``, ``XDP_SKB`` and ``XDP_DRV`` eBPF hooks, based on your preference, at run-time.
+The ``XDP_DRV``, which can guarantee the best performance, can be used only if it is supported by your NIC driver.
 
-The span mode when activated shows all the traffic seen by the service also to the namespace.
- - To activate the span mode the command used is "``polycubectl <cubename> set span=true``".
+To create a service cube and select the desired hook point, just use this command:
 
-Span mode is very useful for debugging; On a shadow cube in span mode programs such as Wireshark or Tcpdump can sniff the traffic.
+::
+
+  polycubectl <service name> add <cube name> type=<hook point>
+
+
+For example if you want to create an instance of the router service and attach it to the XDP_SKB hook point:
```

Comment:
```suggestion
For example if you want to create an instance of the router service working with the XDP_SKB hook point:
```
I would avoid the word _attach_ since no program is attached to a netdevice on cube instantiaton. That word could be confusing.